src/vivarium_gates_nutrition_optimization_child/data/loader.py

```diff
@@ -975,9 +980,16 @@ def load_sam_treatment_rr(key: str, location: str) -> pd.DataFrame:
     #       = (1 - sam_tx_efficacy) * (r2_ux) / (1 - sam_tx_efficacy_tmrel) * (r2_ux)
     #       = (1 - sam_tx_efficacy) / (1 - sam_tx_efficacy_tmrel)
     rr_sam_untreated_remission = (1 - sam_tx_efficacy) / (1 - sam_tx_efficacy_tmrel)
-    rr_sam_untreated_remission[
+
+    better_mam_rows = rr_sam_untreated_remission.copy()
```

Comment:
So this is simply duplicating rr_sam_untreated_remission and renaming the affected entities? i.e. you're just doubling the size of the dataframe with idential values? (same question regarding rr, below)

Reply:
Doubling only the rows involving the MAM state but otherwise correct!